source/manual/fall-back-to-mirror.html.md

```diff
@@ -94,9 +91,9 @@ HTML, it's hard to make broad changes to the site (like putting a banner on ever
 
 You'll be notified by the escalation on-call contact that you need to edit the site.
 
-1. If you're at home, connect to the [VPN][gds-vpn]
+1. If you're at home, connect to the [VPN][gds-vpn].
```

Comment:
These are all numbered 1, deliberate?

Reply:
Yep, they'll get automagically worked out

Reply:
Yes, it's a feature in markdown where you number each item as `1.` then it gets numbered correctly when rendered.  Reduces the likelihood of the list getting out of order when items are added/removed.